### PR TITLE
remove unnecessary observers; stop saving out hdf

### PR DESCRIPTION
**Pull request by stevebachmeier**
### Description

- *Category*:  misc
- *JIRA issue*: na
- *Research reference*:  remove unnecessary observers; stop saving out hdf

### Changes and notes
 I forgot to remove the  unnecessary observers for the timing runs; doing that now.

I also found that there's a bug and the transition_count.hdf is somehow corrupted
and unable to be opened (and I suspect this has been a bug for a very long time;
I confirmed it exists in the oldest data we have which is back in August). I confirmed
with Syl that she doesn't ever use the hdfs so I just stopped saving them out.

### CODE REVIEW

File: vivarium_cluster_tools_launcher_6i5xd_sj.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (mattkappel) [inline vivarium_cluster_tools_launcher_6i5xd_sj.sh]: Mistaken inclusion?

File: src/vivarium_nih_us_cvd/results_processing/process_results.py [hunk 2]
```diff
@@ -57,17 +57,17 @@ class MeasureData(NamedTuple):
...
-    intervention_person_time: pd.DataFrame
+    # healthcare_visits: pd.DataFrame
+    # sbp_medication_person_time: pd.DataFrame
+    # ldlc_medication_person_time: pd.DataFrame
+    # intervention_person_time: pd.DataFrame
 
     def dump(self, output_dir: Path):
         for key, df in self._asdict().items():
-            df.to_hdf(output_dir / f"{key}.hdf", key=key)
+            # df.to_hdf(output_dir / f"{key}.hdf", key=key)
```
Comment (mattkappel) [inline src/vivarium_nih_us_cvd/results_processing/process_results.py:70]: Is this broken or are hdf files not desired (for now)?

**Review by mattkappel — APPROVED** [review]
Looks good except for the VCT temp script in there.

File: src/vivarium_nih_us_cvd/results_processing/process_results.py (hunk 2 above)
Comment (stevebachmeier) [reply src/vivarium_nih_us_cvd/results_processing/process_results.py:70]: Both. transition_count.hdf is and has always been broken (and Hussain has confirmed he sees it as well in mnch). Syl also confirmed she never reads the hdfs.

**Pull request merged**